client/js/constants.js

```diff
@@ -32,11 +32,13 @@ const commands = [
 	"/cs",
 	"/ctcp",
 	"/cycle",
+	"/dehop",
 	"/deop",
 	"/devoice",
 	"/disconnect",
 	"/expand",
 	"/ho",
+	"/hop",
```

Comment:
Is `/ho` intentional? Never paid attention to it, and it's not in the help page.

Reply:
I'm not sure actually. There are a few commands in here that are pass-through commands that we don't do anything with (`as`, `bs`, even `join`) so it could be a weird specific one like that?